pyproject.toml

```diff
@@ -19,6 +19,8 @@ dependencies = [
     "opencv-python-headless >=4.6.0.66",
     "asdf >=2.15.0",
     "gwcs >= 0.18.1",
+    "tweakwcs >=0.8.6",
```

Comment:
Would you add these to the devdeps?
https://github.com/spacetelescope/stcal/blob/eb8528c21e0ffbbdec89f5e1d7323425850d3b69/tox.ini#L56